### PR TITLE
[ISSUE #298] Implement rocketmq connect runtime

### PR DESCRIPTION
**Pull request by odbozhou**
## What is the purpose of the change

https://github.com/apache/rocketmq-externals/issues/298
Implement rocketmq connect runtime

## Verifying this change

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: rocketmq-connect-runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    specific language governing permissions and limitations
+    under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.sonatype.oss</groupId>
+        <artifactId>oss-parent</artifactId>
+        <version>9</version>
```
Comment (zongtanghu) [inline rocketmq-connect-runtime/pom.xml]: Are these element attributes necessary?

**Comment by zongtanghu** [comment]
I review the pr on the whole.Because the pr is too large,we can review every detail of pr's codes.So,I think we can merge this pr firstly,and then we optimize the codes later. @odbozhou @duhenglucky

File: rocketmq-connect-runtime/pom.xml (hunk 1 above)
Comment (odbozhou) [reply rocketmq-connect-runtime/pom.xml]: It has been modified

**Comment by zongtanghu** [comment]
As said yesterday,you can verify with each other by running activemq connector on rocketmq runtime, @odbozhou @githublaohu

**Comment by odbozhou** [comment]
> I test runtime. but startup is error
> org.apache.rocketmq.connect.runtime.exception.RocketMQRuntimeException: Consumer Group is necessary for RocketMQ, please set it.
> 
> The config in connect.conf is miss Group ？

It has been fixed

**Pull request merged**